eledmac.dtx

```diff
@@ -10838,7 +10837,7 @@
 \fi
 %    \end{macrocode}
 % \subsubsection{Options for familiar notes}
-% Before generating the options commands for familiar notes, we check the \verb+nofamiliar+ option.
+% Before generating the optional commands for familiar notes, we check the \verb+nofamiliar+ option.
```

Comment:
On Sun, Apr 12, 2015 at 03:08:41PM -0700, Ma�eul Rouquette wrote:
> > @@ -10838,7 +10837,7 @@
> >  \fi
> >  %    \end{macrocode}
> >  % \subsubsection{Options for familiar notes}
> > -% Before generating the options commands for familiar notes, we check the \verb+nofamiliar+ option.
> > +% Before generating the optional commands for familiar notes, we check the \verb+nofamiliar+ option.
> 
> quelque soit la solution retenue, il faudrait appliquer aussi � la ligne 10866 ("Before generating the options commands for endnotes")

Oui. Et d�finitivement, la bonne phrase est: "before generating the
commands that are used to set the familiar notes/critical notes".

Je me demande m�me s'il ne serait pas bon d'ajouter "such as
\numberonlyfirstinline{} \lemmaseparator{} and the like" pour que tout
soit parfaitement clair dans l'esprit du lecteur.

C'est que c'est tellement technique que cela devient vite tr�s
abstrait.